Tasks/DeleteFilesV1/deletefiles.ts

```diff
@@ -28,7 +30,7 @@ tl.setResourcePath(path.join(__dirname, 'task.json'));
     }
 
     // find all files
-    let foundPaths = tl.find(sourceFolder);
+    let foundPaths: string[] = tl.find(sourceFolder);
```

Comment:
if the `conents` input is `**/*`, then won't the source folder be removed already today?

Reply:
No. The path would then be `/path/to/sourceFolder/**/*` and it would remove everything under `/path/to/sourceFolder/`. We want to also be able to remove `sourceFolder/` itself.